project/Dependencies.scala

```diff
@@ -28,5 +28,5 @@ object Dependencies {
     "com.vladsch.flexmark" % "flexmark-ext-yaml-front-matter" % flexmarkVersion,
   )
 
-  val compilerInterface = "org.scala-sbt" % "compiler-interface" % "1.9.6"
+  val compilerInterface = "org.scala-sbt" % "compiler-interface" % "1.10.4"
```

Comment:
why not `1.10.5` and not `1.10.6` here ?

Reply:
it's intentional. 1.10.4 is the newest version. there isn't always a new compiler-interface for every new sbt version